addons/binding/org.openhab.binding.omnikinverter/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: OmnikInverter Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.omnikinverter;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Export-Package: 
+ org.openhab.binding.omnikinverter,
+ org.openhab.binding.omnikinverter.handler
```

Comment:
We no longer export binding packages by default. 
So please remove the following:

```
Export-Package: 
 org.openhab.binding.omnikinverter,
 org.openhab.binding.omnikinverter.handler
```

and move the classes from these packages into internal packages instead:

* org.openhab.binding.omnikinverter.internal
* org.openhab.binding.omnikinverter.internal.handler

Reply:
File is completely removed.